src/rest-client.ts

```diff
@@ -376,8 +376,8 @@ export class RestClient {
     return this.requestWrapper.get(`spot_margin/market_info${suffix}`);
   }
 
-  getBorrowHistory(): GenericAPIResponse {
-    return this.requestWrapper.get(`spot_margin/borrow_history`);
+  getBorrowHistory(params:BorrowHistoryParams): GenericAPIResponse {
```

Comment:
Thanks for contributing! You can actually use the `TimeRange` interface here:
```suggestion
  getBorrowHistory(params: TimeRange): GenericAPIResponse {
```

It's already used by a few endpoints and shares the same properties (both optional).